utils/getters.js

```diff
@@ -47,3 +47,11 @@ export const getLocalizedArticle = (article, lang) => {
 
 export const getLocalizedArticles = (articles, lang) =>
   articles.map(article => getLocalizedArticle(article, lang));
+
+export const getArticlesRows = (articles, rowSize) => {
```

Comment:
check out [`lodash@chunk`](https://lodash.com/docs/4.17.4#chunk)

Reply:
thanks! I will definitely use it